gunicorn/arbiter.py

```diff
@@ -509,6 +517,8 @@ def spawn_worker(self):
             util._setproctitle("worker [%s]" % self.proc_name)
             self.log.info("Booting worker with pid: %s", worker_pid)
             self.cfg.post_fork(self, worker)
+            if self.num_workers > 1 and worker_id != -1:
```

Comment:
here I would test if the worker id has been set. and then set the env. Although in which case the worker_id couldn't be set there?